package.json

```diff
@@ -64,12 +64,14 @@
     "mocha": "^5.2.0",
     "node-dir": "^0.1.17",
     "npm-run-all": "^4.1.3",
+    "spawn-sync": "^2.0.0",
     "tempy": "^0.2.1",
     "ts-jest": "^22.4.6",
```

Comment:
We should consider bumping this as well, it has a peer dependency on TypeScript 2.x